docs/sql-performance-tuning.md

```diff
@@ -228,6 +228,8 @@ The "REPARTITION_BY_RANGE" hint must have column names and a partition number is
     SELECT /*+ REPARTITION */ * FROM t
     SELECT /*+ REPARTITION_BY_RANGE(c) */ * FROM t
     SELECT /*+ REPARTITION_BY_RANGE(3, c) */ * FROM t
+    SELECT /*+ REBALANCE_PARTITIONS */ * FROM t
```

Comment:
What about just `REBALANCE`? I think that's good enough (vs `COALESCE`).

cc @maryannxue too do you have any suggestion on the name?

Reply:
ah, sounds good to me, these hints are all have the implicit meaning that the effective target is partition. cc @viirya @cloud-fan

Reply:
SGTM. The logical plan name can still be `RebalancePartitions` though.